packages/cli/src/evaluation/test-runner/test-runner.service.ee.ts

```diff
@@ -177,6 +187,12 @@ export class TestRunnerService {
 
 		// 2. Run over all the test cases
 
+		testRun.status = 'running';
+		testRun.runAt = new Date();
+		await this.testRunRepository.save(testRun);
```

Comment:
This is very core business logic, and it would be nice to have it encapsulated somewhere as `markAsRunning` method/function. One option would be in the `TestRunRepository`